compiler+runtime/src/cpp/jank/runtime/obj/lazy_sequence.cpp

```diff
@@ -62,18 +70,16 @@ namespace jank::runtime::obj
 
   lazy_sequence_ptr lazy_sequence::next_in_place()
   {
-    resolve_seq();
-    if(sequence)
+    // sequence is non-nullptr
```

Comment:
Comments are documentation, so please always use `/* */` and proper capitalization and grammar. See the deleted comment on line 68 for an example.